tests/haskell/GhcModBearTest.py

```diff
@@ -0,0 +1,57 @@
+from bears.haskell.GhcModBear import GhcModBear
+from coalib.testing.LocalBearTestHelper import verify_local_bear
+
+# A simple hello world program!
+hello_world_file = """
+main :: IO()
+main = putStrLn "Hello World!"
+"""
+
+# No instance for (Num String) in the first argument of ‘putStrLn’
+not_string_3 = """
+main :: IO()
+main = putStrLn 3
+"""
+
+# Takes a file and prints the number of lines
+count_lines = """
+main :: IO()
+main = interact lineC
+    where lineC input = show (length(lines input))
+"""
+
+# Parse Error (mismatched brackets)
+count_lines_bracket_missing = """
+main :: IO()
+main = interact lineC
+    where lineC input = show (length(lines input)
+"""
+
+# Prints "asd" which is the last but one element in the list
+last_but_one_list = """
+lastButOne :: [a] -> a
+lastButOne xs | (length xs >= 2) = xs !! ((length xs) - 2)
+              | otherwise = error "Not possible"
+
+main :: IO()
+main = do
+    print(lastButOne(["asd", "ASd"]))
+"""
+
+# Couldn't match expected type ‘[a]’ with actual type ‘a’
+last_but_one_list_type_error = """
+lastButOne :: a -> a
+lastButOne xs | (length xs >= 2) = xs !! ((length xs) - 2)
+              | otherwise = error "Not possible"
+
+main :: IO()
+main = do
+    print(lastButOne(["asd", "ASd"]))
+"""
+
+GhcModBearTest = verify_local_bear(
+    GhcModBear,
+    valid_files=(hello_world_file, last_but_one_list, count_lines),
+    invalid_files=(not_string_3, count_lines_bracket_missing,
+                   last_but_one_list_type_error),
+    tempfile_kwargs={'suffix': '.hs'})
```

Comment:
@sils Yeah that's a good idea!!
Can we do that via another issue, since it's just adding the testcase? I'll take it up myself, if no one does in probably a day or so :)